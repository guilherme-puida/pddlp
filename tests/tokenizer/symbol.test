:action
:action-costs
:adl
:condition
:conditional-effects
:constants
:constraints
:continuous-effects
:derived
:derived-predicates
:disjunctive-preconditions
:domain
:domain-axioms ; logistics-round-1-adl
:duration
:duration-inequalities
:durative-action
:durative-actions
:effect
:equality
:existential-preconditions
:fluents
:functions
:goal
:goal-utilities ; woodworking-net-benefit-optimal-numeric-fluents
:init
:length
:metric
:negative-preconditions
:numeric-fluents
:objects
:parallel
:parameters
:precondition
:predicates
:preferences
:quantified-preconditions
:requirements
:serial
:strips
:timed-initial-literals
:types
:typing
:universal-preconditions
:vars ; mystery-prime-round-1-adl
