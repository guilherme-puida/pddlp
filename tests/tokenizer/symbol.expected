[01:01] PDDLP_TOKEN_SYM_ACTION :action
[02:01] PDDLP_TOKEN_SYM_ACTION_COSTS :action-costs
[03:01] PDDLP_TOKEN_SYM_ADL :adl
[04:01] PDDLP_TOKEN_SYM_CONDITION :condition
[05:01] PDDLP_TOKEN_SYM_CONDITIONAL_EFFECTS :conditional-effects
[06:01] PDDLP_TOKEN_SYM_CONSTANTS :constants
[07:01] PDDLP_TOKEN_SYM_CONSTRAINTS :constraints
[08:01] PDDLP_TOKEN_SYM_CONTINUOUS_EFFECTS :continuous-effects
[09:01] PDDLP_TOKEN_SYM_DERIVED :derived
[10:01] PDDLP_TOKEN_SYM_DERIVED_PREDICATES :derived-predicates
[11:01] PDDLP_TOKEN_SYM_DISJUNCTIVE_PRECONDITIONS :disjunctive-preconditions
[12:01] PDDLP_TOKEN_SYM_DOMAIN :domain
[13:01] PDDLP_TOKEN_SYM_DOMAIN_AXIOMS :domain-axioms
[14:01] PDDLP_TOKEN_SYM_DURATION :duration
[15:01] PDDLP_TOKEN_SYM_DURATION_INEQUALITIES :duration-inequalities
[16:01] PDDLP_TOKEN_SYM_DURATIVE_ACTION :durative-action
[17:01] PDDLP_TOKEN_SYM_DURATIVE_ACTIONS :durative-actions
[18:01] PDDLP_TOKEN_SYM_EFFECT :effect
[19:01] PDDLP_TOKEN_SYM_EQUALITY :equality
[20:01] PDDLP_TOKEN_SYM_EXISTENTIAL_PRECONDITIONS :existential-preconditions
[21:01] PDDLP_TOKEN_SYM_FLUENTS :fluents
[22:01] PDDLP_TOKEN_SYM_FUNCTIONS :functions
[23:01] PDDLP_TOKEN_SYM_GOAL :goal
[24:01] PDDLP_TOKEN_SYM_GOAL_UTILITIES :goal-utilities
[25:01] PDDLP_TOKEN_SYM_INIT :init
[26:01] PDDLP_TOKEN_SYM_LENGHT :length
[27:01] PDDLP_TOKEN_SYM_METRIC :metric
[28:01] PDDLP_TOKEN_SYM_NEGATIVE_PRECONDITIONS :negative-preconditions
[29:01] PDDLP_TOKEN_SYM_NUMERIC_FLUENTS :numeric-fluents
[30:01] PDDLP_TOKEN_SYM_OBJECTS :objects
[31:01] PDDLP_TOKEN_SYM_PARALLEL :parallel
[32:01] PDDLP_TOKEN_SYM_PARAMETERS :parameters
[33:01] PDDLP_TOKEN_SYM_PRECONDITION :precondition
[34:01] PDDLP_TOKEN_SYM_PREDICATES :predicates
[35:01] PDDLP_TOKEN_SYM_PREFERENCES :preferences
[36:01] PDDLP_TOKEN_SYM_QUANTIFIED_PRECONDITIONS :quantified-preconditions
[37:01] PDDLP_TOKEN_SYM_REQUIREMENTS :requirements
[38:01] PDDLP_TOKEN_SYM_SERIAL :serial
[39:01] PDDLP_TOKEN_SYM_STRIPS :strips
[40:01] PDDLP_TOKEN_SYM_TIMED_INITIAL_LITERALS :timed-initial-literals
[41:01] PDDLP_TOKEN_SYM_TYPES :types
[42:01] PDDLP_TOKEN_SYM_TYPING :typing
[43:01] PDDLP_TOKEN_SYM_UNIVERSAL_PRECONDITIONS :universal-preconditions
[44:01] PDDLP_TOKEN_SYM_VARS :vars
EOF